src/tutorials/code/node/constellation/1.js

```diff
@@ -0,0 +1,6 @@
+const Carina = require('carina').Carina;
+const ws = require('ws');
+
+Carina.WebSocket = ws;
```

Comment:
we could probably feature-detect here (within Carina) making `ws` an optional dependency...